modules/library/jdbc/src/main/java/org/geotools/jdbc/JDBCFeatureReader.java

```diff
@@ -704,7 +704,7 @@ public boolean isDirty(int index) {
             return dirty[index];
         }
 
-        public boolean isDirrty(String name) {
+        public boolean isDirty(String name) {
```

Comment:
Given the method is public, it cannot be renamed as it would break clients. Instead, create a new method with the right name and deprecate the old one, making it delegate to the new one.